docker-compose.yml

```diff
@@ -6,8 +6,10 @@ services:
       args:
         SKIP_TESTS: "true"
     ports:
-      - "127.0.0.1:80:80"
+      - "127.0.0.1:8080:80"
```

Comment:
I'll put it back to port 80, for me locally it fails to bind to port 80 due to some acces permissions issue. I changed it to 8080 to get it working quickly rather than look into it, and accidentally committed it.